### PR TITLE
Lint ts semicolons (aka. The great semicolon migration)

### PR DESCRIPTION
**Pull request by JorikSchellekens**
I got fed up of missing interface semicolons. So I've altered the tsconfig to be more strict and linted a tonne of files. No functional changes though, should be quick to review even with the 35 files changed

### CODE REVIEW

File: src/components/views/elements/Field.tsx [hunk 1]
```diff
@@ -85,20 +85,20 @@ interface ITextareaProps extends IProps, TextareaHTMLAttributes<HTMLTextAreaElem
...
+    feedbackVisible: boolean;
+    focused: boolean;
 }
 
 export default class Field extends React.PureComponent<PropShapes, IState> {
     private id: string;
     private input: HTMLInputElement;
 
-    private static defaultProps = {
+    public static readonly defaultProps = {
```
Comment (JorikSchellekens) [inline src/components/views/elements/Field.tsx:98]: I'm aware of this and I thought I could sneak in a tiny extra lint while I was at it

File: src/components/views/elements/Slider.tsx [hunk 2]
```diff
@@ -86,8 +86,8 @@ export default class Slider extends React.Component<IProps> {
             const offset = this.offset(this.props.values, this.props.value);
             selection = <div className="mx_Slider_selection">
                 <div className="mx_Slider_selectionDot" style={{left: "calc(-0.55em + " + offset + "%)"}} />
-                <hr style={{width: offset + "%"}} />
-            </div>
+                <hr style={{width: offset + "%;"}} />
```
Comment (t3chguy) [inline src/components/views/elements/Slider.tsx:89]: this looks wrong

Comment (turt2live) [reply src/components/views/elements/Slider.tsx:89]: the indentation is right though

Comment (JorikSchellekens) [reply src/components/views/elements/Slider.tsx:89]: Yeah, that's something else I've been trying to resolve:
image

Comment (JorikSchellekens) [reply src/components/views/elements/Slider.tsx]: Removed for now

**Pull request merged**